src/quicr_server_raw_session.cpp

```diff
@@ -324,12 +320,11 @@ ServerRawSession::handle_publish_intent(
   } else {
     auto state = publish_namespaces[intent.quicr_namespace].state;
     switch (state) {
+      // TODO(trigaux): Resend response?
       case PublishIntentContext::State::Pending:
-        // TODO: Resend response?
-        break;
+      // TODO(trigaux): Already registered this namespace successfully, do
+      // nothing?
       case PublishIntentContext::State::Ready:
-        // TODO: Already registered this namespace successfully, do nothing?
-        break;
       default:
         break;
     }
```

Comment:
Surprised compiler didn't complain about fallthrough:
```cpp
      case PublishIntentContext::State::Pending:
        [[fallthrough]] // TODO: Resend response?
      case PublishIntentContext::State::Ready:
        [[fallthrough]] // TODO: Already registered this namespace successfully, do nothing?
      default:
        break;

Reply:
clang-tidy complained about identical branches, which it still does even with the `[[fallthrough]]` annotation.  But I `NOLINT`'ed it.